frontend/src/metabase/visualizations/echarts/cartesian/constants/style.ts

```diff
@@ -4,6 +4,7 @@ export const CHART_STYLE = {
     zIndexLineArea: 3, // https://github.com/metabase/metabase/issues/40209
     barWidth: 0.8,
     histogramBarWidth: 0.995,
+    barMinWidth: 27,
```

Comment:
@alxnddr do you know if we ever had a value for that before echarts?
It's just the narrowest value I could achieve locally on `master`, so not sure if it's correct

Reply:
@kulyk I believe there should not be min bar width on bars and even 1px is a valid width
<img width="1356" alt="Screenshot 2024-04-02 at 12 37 11 PM" src="https://github.com/metabase/metabase/assets/14301985/86c38f57-103a-4f14-86c5-d38ff76e118b">

I believe the original issue is about weird resizing which is especially noticeable on dashboards. You can open the [staging](https://metabase-echarts.hosted.staging.metabase.com/dashboard/1887) you can notice it initially renders with proper dimensions and then weirdly resizes so that there are incorrect margins